.travis.yml

```diff
@@ -33,17 +35,21 @@ notifications:
 
 # command to run tests
 script:
-  - python3 -m imapfw.edmp
-  - python3 -m imapfw.mmp.manager
-  - ./imapfw.py -h
-  - ./imapfw.py noop
-  - ./imapfw.py -c multiprocessing unitTests
-  - ./imapfw.py -c threading unitTests
-  - ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all noop
-  - ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all testRascal
-  - ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all examine
-  - ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c multiprocessing syncAccounts -a AccountA
-  - ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c threading syncAccounts -a AccountA
-  - ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c threading syncAccounts -a AccountA -a AccountA -a AccountA
-
+  - coverage run --source=imapfw,imapfw.py -p -m imapfw.edmp
+  - coverage run --source=imapfw,imapfw.py -p -m imapfw.mmp.manager
+  - coverage run --source=imapfw,imapfw.py -p ./imapfw.py -h
+  - coverage run --source=imapfw,imapfw.py -p ./imapfw.py noop
+  - coverage run --source=imapfw,imapfw.py -p ./imapfw.py -c multiprocessing unitTests
+  - coverage run --source=imapfw,imapfw.py -p ./imapfw.py -c threading unitTests
+  - coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all noop
+  - coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all testRascal
+  - coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all examine
+  - coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c multiprocessing syncAccounts -a AccountA
+  - coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c threading syncAccounts -a AccountA
+  - coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c threading syncAccounts -a AccountA -a AccountA -a AccountA
+  - coverage combine .coverage*
```

Comment:
It runs the python version of the current virtual environment. If it is python2.X, it will run it in python2.X (and fail). But the changes are meant for travis anyway, and python3.X is enforced earlier in the configuration file so it won't happen.